iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/TestAbstractVoltageLevel.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```

Comment:
Sorry for unresolving, but in fact after asking a colleague more experienced in that matter, it should rather be your name directly instead of "powsybl authors". Plus it doesn't really make sense to put "Powsybl authors" only in this file and not in the others...

Reply:
No worries, fixed!